src/lib/y2storage/blk_device.rb

```diff
@@ -682,6 +698,15 @@ def volume_match_values
       }
     end
 
+    def available_mount_bys
```

Comment:
This method is kind of temporary. I will now open a pull request against libstorage-ng to make `BlkDeviceImpl#possible_mount_bys` public. That would make more sense than duplicating logic here.

Reply:
This is the due pull request https://github.com/openSUSE/libstorage-ng/pull/713

Reply:
libstorage-ng PR already merged.